<launch>
	<include file="$(find dave_demo)/bandit_alt.launch"/>

	<node pkg="pose_action" type="pose_action_server" name="pose_action_server">
		
	</node>

	<node pkg="wait_for_response_action" type="wait_for_response_server" name="wait_for_response_server"/>
	
	<!--<node pkg="simon_says" type="simon_says" name="simon_says"/>-->
</launch>
